### DISCUSSION TITLE
如何对jinja模板文件增加高亮显示

### DISCUSSION THREAD

**Discussion by fishtailstudio** [izhangzhihao/intellij-rainbow-brackets#2700] [Q&A] [answered]
我看这个插件的介绍里说支持jinja，但是我在Clion、RustCover上使用了这个插件，均没有高亮。

**Comment by izhangzhihao** [comment, marked as answer]
hi, [Clion does not support Jinja](https://intellij-support.jetbrains.com/hc/en-us/community/posts/206594815/comments/4414002942098), you can try using Pycharm Pro or Intellij Ultimate version.

**Reply by fishtailstudio** [reply]
Ok, thanks.

**Comment by izhangzhihao** [comment]
Or you can follow: https://youtrack.jetbrains.com/issue/CPP-13470
